Utilities/DataLoading/EdFi.LoadTools/Engine/Factories/JsonMetadataFactory.cs

```diff
@@ -20,15 +20,19 @@ public JsonMetadataFactory(SwaggerMetadataRetriever swaggerMetadata)
 
         public IEnumerable<JsonModelMetadata> GetMetadata()
         {
-            return _swaggerMetadata.GetMetadata().Result
-                                   .Where(
-                                        j => j.Property != "_etag"
-                                             && j.Property != "id"
-                                             && j.Property != "link"
-                                             && j.Property != "priorDescriptorId"
-                                             && !(j.Model.EndsWith("Descriptor") && j.Property == $"{j.Model}Id")
-                                    )
-                                   .Distinct(new ModelMetadataEqualityComparer<JsonModelMetadata>());
+            var task = _swaggerMetadata.GetMetadata().ConfigureAwait(false);
```

Comment:
This just needs to be made async, not more hacks in for trying to force it synchronous.

Reply:
ODS-4721